srcpkgs/xfsprogs/template

```diff
@@ -1,17 +1,17 @@
 # Template file for 'xfsprogs'
 pkgname=xfsprogs
-version=5.13.0
-revision=2
+version=5.18.0
+revision=1
 build_style=gnu-configure
 configure_args="--enable-editline=yes --enable-lib64=no --libdir=/usr/lib"
 hostmakedepends="gettext libuuid-devel"
-makedepends="libblkid-devel libedit-devel inih-devel"
+makedepends="libblkid-devel libedit-devel inih-devel liburcu-devel"
 short_desc="Utilities for managing the XFS filesystem"
 maintainer="Enno Boland <gottox@voidlinux.org>"
 license="LGPL-2.1-or-later"
 homepage="http://xfs.org/index.php/Main_Page"
 distfiles="${KERNEL_SITE}/utils/fs/xfs/xfsprogs/xfsprogs-${version}.tar.xz"
-checksum=4e142d4babe086adf9016d8c606c805829da08e46389a4433f40346204f90cdb
+checksum=1e8d8801bdec8cd4cad360ce3bbd12c35a97f2bc8f7c8c9580d1903b0e8cc35b
 
 post_extract() {
```

Comment:
Please remove this block

Reply:
Assuming you meant the post_extract block, done tried without with and it compiles fine ! 

Tested on a vm with a xfs root and it works fine